usaspending_api/references/v2/views/recipients.py

```diff
@@ -86,9 +86,20 @@ def _create_es_search(self, search_text: str, recipient_levels: List[str], limit
             An Elasticsearch search query.
         """
         es_recipient_search_fields = ["recipient_name", "uei", "duns"]
-        query_string_query = ES_Q("query_string", query=f"*{search_text}*", fields=es_recipient_search_fields)
-        multi_match_query = ES_Q("multi_match", query=search_text, fields=es_recipient_search_fields)
-        query = ES_Q("bool", should=[query_string_query, multi_match_query], minimum_should_match=1)
+
+        should_query = []
+        for search_field in es_recipient_search_fields:
+            should_query.append(
+                ES_Q("match_phrase_prefix", **{f"{search_field}": {"query": search_text, "boost": 5}}),
+            )
+            should_query.append(
+                ES_Q("match_phrase_prefix", **{f"{search_field}.contains": {"query": search_text, "boost": 3}}),
+            )
+            should_query.append(
+                ES_Q("match", **{f"{search_field}": {"query": search_text, "operator": "and", "boost": 1}})
+            )
```

Comment:
This is a nit, but this could be done in a list comprehension. 

```suggestion
        should_query = [
            query
            for search_field in es_recipient_search_fields
            for query in [
                ES_Q("match_phrase_prefix", **{f"{search_field}": {"query": search_text, "boost": 5}}),
                ES_Q("match_phrase_prefix", **{f"{search_field}.contains": {"query": search_text, "boost": 3}}),
                ES_Q("match", **{f"{search_field}": {"query": search_text, "operator": "and", "boost": 1}}),
            ]
        ]
```
```

Reply:
I originally had this, but didn't like the way it looked. I've added it back, because it's the more concise way of building the queries.